src/TMDBApp.Core/Services/RequestsProvider/RequestProvider.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using TMDBApp.Core.Exception;
+using TMDBApp.Core.Factories;
+
+namespace TMDBApp.Core.Services.RequestsProvider
+{
+    public class RequestProvider : IRequestProvider
+    {
+        private readonly JsonSerializerSettings _serializerSettings;
+
+        public RequestProvider()
+        {
+            _serializerSettings = new JsonSerializerSettings
+            {
+                ContractResolver = new CamelCasePropertyNamesContractResolver(),
```

Comment:
Could have used `SnakeCaseNamingStrategy()` to avoid creating annotations on models.